tests/tests_passing_pylint.txt

```diff
@@ -1,5 +1,4 @@
 tests/data/**
-tests/devices/qubit/test_apply_operation.py
```

Comment:
(tests/devices/qubit/** is already in the file below, I'm not just deleting this 😋)